tabs/tabs/src/androidTest/java/com/microsoft/device/dualscreen/tabs/SurfaceDuoTabLayoutTest.kt

```diff
@@ -35,66 +36,75 @@ import org.junit.runner.RunWith
 class SurfaceDuoTabLayoutTest {
 
     @get:Rule
-    val activityTestRule = ActivityTestRule(SimpleTabActivity::class.java)
+    val activityTestRule = ActivityScenarioRule(SimpleTabActivity::class.java)
+    private val uiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+
+    val rule = createWindowLayoutInfoPublisherRule()
 
     @Test
     fun testDisplayPositionFromLayout() {
-        SurfaceDuo1.switchFromSingleToDualScreen()
+        uiDevice.spanFromStart()
+        onView(withId(R.id.tabs)).check(matches(areTabsOnScreen(DisplayPosition.DUAL)))
+    }
+
+    @Test
+    fun testCesar() {
```

Comment:
I hope this test never fails 😄 should we rename it?

Reply:
ouchhhhhhhhhh 😅